src/docs/Foundations/Layout/Grid/index.mdx

```diff
@@ -28,34 +25,34 @@ Those properties can change depending on the breakpoint.
 When using a grid, you decide the size of your elements by **the number of columns they use.**
 
 <HintItem>
-  An element should always stand on one or between several columns
+  An element should always stand on one or between several columns.
 </HintItem>
 <HintItem>
-  Elements are always separated by either a gutter or a combination of columns
+  Elements are always separated by either a gutter or a combination of columns.
   and gutter
 </HintItem>
-<HintItem dont>An element should never start on a gutter</HintItem>
+<HintItem dont>An element should never start on a gutter.</HintItem>
 <br />
 
 ![GridPossibilities](GridPossibilities.png)
 ![GridSpacing](GridSpacing.png)
 
-# The Leroy Merlin default Grid
+# LEROY MERLIN's default grid
```

Comment:
Why use capital letters?

Reply:
It's in the brand guidelines, LEROY MERLIN must always be written in capital letters.